{{cookiecutter.project_slug}}/binder/postBuild

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
-jupyter serverextension enable --py nbserverproxy --sys-prefix
+jupyter serverextension enable --py jupyter_server_proxy --sys-prefix
```

Comment:
From what I can tell in the docs, this shouldn't be necessary when installing from pip or Conda: https://github.com/jupyterhub/jupyter-server-proxy. Does that sound correct?

Reply:
Yes, that sounds right to me.

Reply:
What would be the best/simple example to test this?

Reply:
The package uses a conf.d approach to auto-enable itself. So if you remove this line and things still work, you should be good to go.

Reply:
> and things still work

Yeah, I'm asking what "things" are. I'm not using dask or the dashboards in any of my current binder-based projects. If I can make a dead simple notebook example that uses the extension I was hoping that would be a good test. Or are you saying it should be auto-enabled and if the install line below this succeeds then we can assume it works?